grails-app/conf/Config.groovy

```diff
@@ -86,7 +86,7 @@ downloadAuth {
 }
 
 featureToggles {
-    // someFeature = <boolean>
+    dynamicGeoserverFilters = false
```

Comment:
Per server is the way to go IMO. I'll fix that on the `no_db` branch.

However we might also want to enable filters from plain simple files, it can still be useful!